### PR TITLE
remove z in rbf

### PR DESCRIPTION
**Pull request by ken2403**
The contribution of the atomic number Z to the basis function has been removed.

### CODE REVIEW

File: lcaonet/nn/rbf.py [hunk 1]
```diff
@@ -23,10 +23,15 @@ def __init__(self, cutoff: float | None, atom_info: BaseAtomisticInformation):
         self.n_orb = atom_info.n_orb
         self.n_l_list = atom_info.get_nl_list
 
+    def extra_repr(self) -> str:
```
Comment (ken2403) [inline lcaonet/nn/rbf.py:26]: add repr

File: lcaonet/nn/cutoff.py [hunk 2]
```diff
@@ -42,6 +42,9 @@ def __init__(self, cutoff: float):
         super().__init__()
         self.cutoff = cutoff
 
+    def extra_repr(self) -> str:
```
Comment (ken2403) [inline lcaonet/nn/cutoff.py:45]: add repr

File: lcaonet/nn/rbf.py [hunk 3]
```diff
@@ -49,12 +54,11 @@ def __init__(
         self.basis_func: list[HydrogenRadialWaveFunctionBasis.basis_type] = []
         self.normalize_coeff: dict[tuple[int, int], Tensor] = {}
         for n, l in self.n_l_list:
-            self.normalize_coeff[(n, l)] = torch.ones(self.max_z + 1)
```
Comment (ken2403) [inline lcaonet/nn/rbf.py:52]: remove contribution of z in normalize coefficient

File: lcaonet/model/lcaonet.py [hunk 4]
```diff
@@ -470,16 +464,12 @@ def forward(
...
-        ).unsqueeze(1)
-
         # node update
-        x = x_before + self.node_after_lin(scatter(lcao_w * self.node_lin(torch.cat([xi, xj], dim=-1)), idx_i, dim=0))
+        x = x_before + self.node_after_lin(
+            scatter(lcao_w * self.node_lin(torch.cat([x[idx_i], x[idx_j]], dim=-1)), idx_i, dim=0)
+        )
 
-        return x, cji
+        return x
```
Comment (ken2403) [inline lcaonet/model/lcaonet.py:472]: also, remove updating coeffsient vector
